File: read_pdf.js
const fs = require('fs');
const pdf = require('pdf-parse');
let dataBuffer = fs.readFileSync('c:/Users/Sistemas/Downloads/pruebaantigravity/READMECLE.pdf');
pdf(dataBuffer).then(function (data) {
    fs.writeFileSync('c:/Users/Sistemas/Downloads/pruebaantigravity/pdf_text.txt', data.text, 'utf8');
}).catch(function (err) {
    fs.writeFileSync('c:/Users/Sistemas/Downloads/pruebaantigravity/pdf_err.txt', err.toString(), 'utf8');
});

File: read_pdf.py
import sys
try:
    from pypdf import PdfReader
    reader = PdfReader('c:/Users/Sistemas/Downloads/pruebaantigravity/READMECLE.pdf')
    text = ""
    for page in reader.pages:
        text += page.extract_text() + "\n"
    with open('c:/Users/Sistemas/Downloads/pruebaantigravity/pdf_text_py.txt', 'w', encoding='utf-8') as f:
        f.write(text)
    print("Success")
except Exception as e:
    print(f"Error: {e}")
